CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ release.
 
 ## Unreleased
 
+### Breaking
+
+### Features
+
+### Fixes
+
+## v1.24.0 (2023-12-15)
```

Comment:
The following entries appear to be missing:

- https://github.com/open-telemetry/semantic-conventions/pull/499
- https://github.com/open-telemetry/semantic-conventions/pull/491
- https://github.com/open-telemetry/semantic-conventions/pull/493
- https://github.com/open-telemetry/semantic-conventions/pull/490
- https://github.com/open-telemetry/semantic-conventions/pull/573
- https://github.com/open-telemetry/semantic-conventions/pull/333

Reply:
Thanks @arminru! I did a manual comparison and fixed all the entries I've discovered.